kong/db/declarative/migrations/route_path.lua

```diff
@@ -1,7 +1,6 @@
 local migrate_path = require "kong.db.migrations.migrate_path_280_300"
 
-return function(tbl)
-  local version = tbl._format_version
+return function(tbl, version)
```

Comment:
how about `local version = version or tbl._format_version`

Reply:
> how about `local version = version or tbl._format_version`

We cannot get the version from this table. The version field is removed by the `flatten` call.